src/components/List/_List.scss

```diff
@@ -10,6 +10,53 @@
 .list {
   margin: unset; // browser automatically applies margin
 
+  &__definition-list {
+    border-bottom: 1px solid var(--ui-gray-light);
+
+    h3 {
+      border-top: 3px solid var(--section-research-primary);
+      padding-top: var(--space-xs);
+    }
+
+    dl {
+      display: grid;
+      gap: 0px;
+      grid-template-columns: 100%;
+      grid-template-rows: 1fr;
+      margin-bottom: 0;
+
+      dt {
+        border-top: 1px solid var(--ui-gray-light);
+        font-size: var(--font-size-0);
+        font-weight: bold;
+        padding-bottom: 0;
+        padding-right: var(--space-m);
```

Comment:
Why is padding preferred over margin?  `space-stack` [uses margin](https://github.com/NYPL/nypl-design-system/blob/development/src/styles/03-space/_space-stack.scss) and a lot of things use space-stack, I'm wondering, if padding is a better thing to use, if it might be good to make similar changes to space-stack

Reply:
Tbh, I wasn't aware of the space-stack mixin until you mentioned it!  In this case, I used padding because I needed the space to appear on the inside of the box model otherwise the gray border would have separated.  On the bottom, I used padding for consistency, but if it is preferable to use the mixin, I could switch over to that.  I think @bigfishdesign13 has some thoughts on the mixins that we can talk about in an upcoming meeting as well.

Reply:
Padding vs. Margin depends on the situation.  As Michael noted, for his case padding was necessary to accomplish the look that he needed.  But maybe we need to expand the spacing options because of problems like what Michael was facing.